torsiondrive/tests/test_wq_tools.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test for the wq_tools module.
+"""
+
+import pytest
+import os
+import sys
+import subprocess
+
+@pytest.mark.skipif("work_queue" not in sys.modules, reason='work_queue not found')
+def test_work_queue():
+    from torsiondrive.wq_tools import WorkQueue
+    wq = WorkQueue(56789)
+    wq.submit('echo test > test.txt', [], ['test.txt'])
+    assert wq.get_queue_status() == (0,0,0,0)
+    # submit a worker
+    p = subprocess.Popen("$HOME/opt/cctools/bin/work_queue_worker localhost 56789 -t 1", shell=True)
```

Comment:
I haven't used WorkQueue before, but I wonder if there's a more general way to describe this path. This seems highly system-specific.

Reply:
This path matches the script `devtools/travis-ci/install-cctools-62.sh` we used to install cctools.

Reply:
Ah, I see. That should be safe, then!